dimod/reference/samplers/exact_solver.py

```diff
@@ -159,6 +159,45 @@ def sample_poly(self, polynomial, **kwargs):
         """
         return ExactSolver().sample(polynomial, **kwargs)
 
+class ExactSolverDQM(Sampler):
```

Comment:
Nit - I would call this `ExactDQMSolver` to be consistent with the convention we use for `LeapHybridDQMSampler`

Reply:
And also 'ExactPolySolver'